spec/sentry_debug_files_upload_spec.rb

```diff
@@ -0,0 +1,324 @@
+describe Fastlane do
+  describe Fastlane::FastFile do
```

Comment:
`l`: The spec is almost the same as `sentry_upload_dif_spec`. Again it would be great if we could reuse some code.